openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx

```diff
@@ -486,8 +487,8 @@ export const getGraphDataByTierType = (rawData: TotalEntitiesByTier[]) => {
   });
 
   const graphData = prepareGraphData(timestamps, filteredData);
-  const latestData = getLatestCount(last(graphData));
-  const oldestData = getLatestCount(first(graphData));
+  const latestData = getLatestCount(omit(last(graphData), 'NoTier'));
```

Comment:
localisation

Reply:
Hi @chirag-madlani we are just omitting key here